netuitive/conf/collectors/ProcessCheckCollector.conf

```diff
@@ -1,7 +1,30 @@
+# Enable the collector and configure the processes that need to be monitored. 
+# There are three options for matching a process: process name, process executable or process command line. 
+# exe and name are both lists of comma-separated regexps.
+
+# [process] is the parent and [[example_process_name]] are the child entries.  
```

Comment:
This wording is a little bit awkward. Would something like 

> [process] is the parent and entries like [[example_process_name]] are its children.

 read any better